docs/source/create/avatars/create-avatars.rst

```diff
@@ -32,9 +32,7 @@ The steps involved in creating your avatar are:
 1. Create an avatar with 3D character modeling tool such as Adobe Fuse, Blender or Maya. 
 2. Rig and animate your avatar with an animation tool such as Mixamo.
 3. Fine tune your avatar using a tool such as Blender or Maya.
-4. Package the model in High Fidelity for use as an avatar.  
-
-.. note:: If you intend to upload and sell your avatar to the Marketplace, you need to set your base material color to white (some apps default to grey). This ensures that the avatar renders correctly for all users and that it will be accepted into our Marketplace.
+4. Package the model in Athena for use as an avatar.  
```

Comment:
"Project Athena" instead of "Athena"?
```
4. Package the model in Project Athena for use as an avatar.
```

And elsewhere on the page?

Reply:
Been trying to find places to put the shorthand since we have that luxury that High Fidelity did not. "In Fidelity..." "In High...." LOL so all they had was HiFi.